swap_meet/vendor.py

```diff
@@ -0,0 +1,75 @@
+from swap_meet.item import Item
+
+class Vendor:
+    def __init__(self, inventory=None):
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
+
+    def add(self, new_item):
+        self.inventory.append(new_item)
+        return new_item
+
+    def remove(self, removed_item):
+        for inventory_item in self.inventory:
+            if inventory_item is removed_item:
+                self.inventory.remove(removed_item)
+                return removed_item
+        return False
+
+    def get_by_category(self, category=""):
+        items = []
+        for inventory_item in self.inventory:
+            if(inventory_item.category is category):
```

Comment:
You've probably seen `is` used a lot in our tests, but it's actually more accurate to use `==` to test for equivalency. This is because `is` checks if two variables point to the same object, while `==` checks for equivalency. 

Here is more info: https://dbader.org/blog/difference-between-is-and-equals-in-python